README.md

```diff
@@ -1,8 +1,8 @@
 # MoonDAO.com Website
 
-[![](https://gray-main-toad-36.mypinata.cloud/ipfs/QmZDpmqsxJwk4x53DkvB3vek1nXFn4GJxGCc94fPBpicuA)](https://www.moondao.com)
+[![](https://gray-main-toad-36.mypinata.cloud/ipfs/bafkreifucveibyrj3px25qg7rajl77gkgkrzvtrsno57zo4pbmq3ntazkq)](https://www.moondao.com)
 
-[MoonDAO.com](https://www.moondao.com) is the hub for the Internet's Space Program, where users can become a MoonDAO member, obtain $MOONEY, lock to participate in governance, submit proposals, vote, read news and updates, dive into the documentation, and interact with the Marketplace for unique experiences such as the Ticket to Space sweepstakes or zero gravity astronaut training experiences. To get started, users can connect their Web3 wallet, or create one by email or social login, in order to interact with the MoonDAO smart contracts.
+[MoonDAO.com](https://www.moondao.com) is the hub for the Internet's Space Program, where users can become a MoonDAO Citizen, buy $MOONEY, lock to participate in governance, submit proposals, vote, read news and updates, dive into the documentation, and purchase from the Marketplace with unique experiences such as the Ticket to Space sweepstakes or zero gravity astronaut training experiences and offerings from Teams in the Space Acceleration Network. To get started, users can connect their onchain wallet, or create one by email or social login, in order to interact with the MoonDAO smart contracts.
```

Comment:
I thought we were going with Citizen v.s. MoonDAO Citizen since technically you're a SAN Ctizien v.s. MoonDAO. Or do you think we should we revisit this and start to just call them MoonDAO Citizens?